WebKitBrowser/WebKitBrowser.cpp

```diff
@@ -279,8 +277,13 @@ namespace Plugin {
         if (path.empty() == false) {
             string fullPath = _persistentStoragePath + path;
             Core::Directory dir(fullPath.c_str());
-            if (!dir.Destroy(true)) {
-                TRACE(Trace::Error, (_T("Failed to delete %s\n"), fullPath.c_str()));
+#if defined(THUNDER_VERSION) && THUNDER_VERSION >= 4
```

Comment:
Do we still expect changes on R2 that this is needed? (guess those build will not have updated hashes). 
This #if will conflict with builds that do not have these defines (like ours and other customers). 
At the same time we will look to make the Destroy changes backwards compatible, it should have not been made to brake backwards compatibility I think.

Reply:
Yes, we expect latest browser plugin revision to compile with R2.

Reply:
On the topic of backward compatibility: R4.4 removed Announcements() from the RPC::IIPCServer  interface. Do you know if that was intentional as well? It looks like we'll have to patch browser plugin to remove https://github.com/rdkcentral/rdkservices/blob/sprint/2401/WebKitBrowser/Extension/main.cpp#L81.
```
    PluginHost()
        : _engine(Core::ProxyType<RPC::InvokeServerType<2, 0, 4>>::Create())
        , _comClient(Core::ProxyType<RPC::CommunicatorClient>::Create(GetConnectionNode(), Core::ProxyType<Core::IIPCServer>(_engine)))
    {
        _engine->Announcements(_comClient->Announcement());  /// <<<-----
    }
  ```
 Any suggestions on how to handle it differently (in a way that is backward compatible to R2 and R4.2)?